tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -89,48 +90,105 @@ describe(__filename, () => {
       });
     }
 
-    function getReviewCount(root) {
+    function getReviewData(root) {
       return root.find(MetadataCard).prop('metadata')[1];
     }
 
-    function getAverageMeta(root) {
+    function getReviewTitle(root) {
+      const { title } = getReviewData(root);
+      return shallow(<div>{title}</div>);
+    }
+
+    function getReviewCount(root) {
+      const { content } = getReviewData(root);
+      return shallow(<div>{content}</div>);
+    }
+
+    function getAverageData(root) {
       return root.find(MetadataCard).prop('metadata')[2];
     }
 
-    it('renders a count of multiple ratings', () => {
-      const root = renderRatings({ count: 5 });
+    function getAverageTitle(root) {
+      return shallow(getAverageData(root).title);
+    }
 
-      expect(getReviewCount(root).content).toEqual('5');
-      expect(getReviewCount(root).title).toEqual('Ratings');
-    });
+    function getAverageNumber(root) {
+      return shallow(getAverageData(root).content);
+    }
 
-    it('renders a count of one rating', () => {
-      const root = renderRatings({ count: 1 });
+    it('renders a count of multiple reviews', () => {
+      const slug = 'some-slug';
+      const root = render({
+        addon: createInternalAddon({
+          ...fakeAddon,
+          ratings: { count: 10, text_count: 5 },
+          slug,
+        }),
+      });
 
-      expect(getReviewCount(root).content).toEqual('1');
-      expect(getReviewCount(root).title).toEqual('Rating');
+      expect(getReviewTitle(root).find(Link)).toHaveProp(
+        'to',
+        `/addon/${slug}/reviews/`,
+      );
+      expect(
+        getReviewTitle(root)
+          .find(Link)
+          .children(),
+      ).toHaveText('Reviews');
+      expect(getReviewCount(root).find(Link)).toHaveProp(
+        'to',
+        `/addon/${slug}/reviews/`,
+      );
+      expect(
+        getReviewCount(root)
```

Comment:
I think it would be worth moving up `getReviewCount(root)` and `getReviewTitle(root)` to a shared variable.